sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -586,9 +647,99 @@
+                "condition": "[equals(parameters('setup_materialization_store'), 'true')]",
+                "type": "connections",
+                "apiVersion": "2022-05-01",
+                "name": "[variables('offlineStoreConnectionName')]",
+                "location": "[parameters('location')]",
+                "dependsOn": [
+                    "[resourceId('Microsoft.MachineLearningServices/workspaces', parameters('workspaceName'))]"
+                ],
+                "identity": {
+                    "type": "SystemAssigned"
+                },
+                "properties": {
+                    "category": "AzureDataLakeGen2",
+                    "target": "[parameters('offline_store_connection_target')]",
+                    "authType": "ManagedIdentity",
+                    "credentials": {
+                        "clientid": "[parameters('materialization_identity_client_id')]",
+                        "resourceid": "[parameters('materialization_identity_resource_id')]"
+                    }
+                }
+            }]
         },
+        {
+            "condition": "[equals(parameters('setup_materialization_store'), 'true')]",
+			"type": "Microsoft.Resources/deployments",
+			"apiVersion": "2022-05-01",
+			"name": "[concat(parameters('workspaceName'), '-update-UAI')]",
+			"dependsOn": [
+				"[parameters('workspaceName')]"
+			],
+			"properties": {
+				"mode": "Incremental",
+				"parameters": {},
+				"template": {
+					"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+					"contentVersion": "1.0.0.1",
+					"resources": [{
+						"apiVersion": "2022-01-01-preview",
+						"name": "[parameters('workspaceName')]",
+						"location": "[parameters('location')]",
+						"kind": "featurestore",
+						"type": "Microsoft.MachineLearningServices/workspaces",
+						"identity": {
+							"type": "SystemAssigned,UserAssigned",
+							"userAssignedIdentities": {
+							    "[parameters('materialization_identity_resource_id')]": {}
+							}
+						},
+						"properties": {
+                            "friendlyName": "[parameters('friendlyName')]",
+                            "description": "[parameters('description')]",
+                            "storageAccount": "[variables('storageAccount')]",
+                            "keyVault": "[variables('keyVault')]",
+                            "applicationInsights": "[variables('applicationInsights')]",
+                            "containerRegistry": "[if(not(equals(parameters('containerRegistryOption'), 'none')), variables('containerRegistry'), json('null'))]",
+                            "hbiWorkspace": "[parameters('confidential_data')]",
+                            "imageBuildCompute": "[parameters('imageBuildCompute')]",
+                            "publicNetworkAccess": "[parameters('publicNetworkAccess')]",
+                            "softDeleteEnabled": "[parameters('soft_delete_enabled')]",
```

Comment:
please remove this one. This property is in private preview. Not published in swagger yet.

Reply:
These property are present in the workspace template already, and it is for incremental update that we should have all the property.

Reply:
not used, will remove from incremental update